execute/run.go

```diff
@@ -4,14 +4,17 @@ import (
 	"encoding/json"
 	"fmt"
 	"io/ioutil"
-	"net/http"
 	"net/url"
+	"time"
+
+	"golang.org/x/net/context"
+	"golang.org/x/net/context/ctxhttp"
 
 	"github.com/yarpc/crossdock/plan"
```

Comment:
Nit: Fix import group ordering. It should be:

   std library

    local packages

    third party packages